src/main/java/eu/enexa/service/EnexaServiceImpl.java

```diff
@@ -242,22 +243,34 @@ public Model containerStatus(String experimentIri, String instanceIRI) {
     }
 
     @Override
-    public Model stopContainer(String experimentIri, String containerIri) {
+    public Model stopContainer(String experimentIri, String containerID) {
         Model model = ModelFactory.createDefaultModel();
+        String ResultOfDtoppingTheContainer = containerManager.stopContainer(containerID);
```

Comment:
1. the variable should start lowercased
2. It should be "stopping"

Reply:
done